vimeo-networking/src/test/java/com/vimeo/networking/utils/PrivacySettingsParamsTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.vimeo.networking.utils
+
+import com.vimeo.networking.Vimeo
+import com.vimeo.networking.model.Privacy
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+/**
+ * Tests for {@link PrivacySettingsParams}.
+ *
+ * Created by Mohit Sarveiya on 10/30/15.
+ */
+class PrivacySettingsParamsTest {
+
+    private val privacySettingsParams = PrivacySettingsParams()
```

Comment:
this property could just be a local variable in each test since there does not need to be shared state.